packages/@aws-cdk/cloudformation-include/test/test-templates/fn-sub-shadow-attribute.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Resources": {
+    "Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": {
+          "Fn::Sub": [
+            "${AnotherBucket.DomainName}",
```

Comment:
Did you confirm this is valid in CloudFormation?

Reply:
yes, that template even fully deploys